package.json

```diff
@@ -46,9 +46,8 @@
     "extends": "stylelint-config-terra"
   },
   "scripts": {
-    "clean": "node scripts/clean-modules/index.js",
```

Comment:
Do we want to remove clean entirely or replace it with `rm -rf node_modules`?

Reply:
I opted to remove clean since other terra packages / repos so not have a clean script but I have no preference at all. Want me to keep it?

Reply:
keep it, shorter commands ftw

Reply:
imo we should replace 'clean:all' in all other terra repos with 'clean'

Reply:
lets do that then

Reply:
@StephenEsser  @mjhenkes 

Thoughts on `clean` script removes node_modules
and then `precompile` script for removing lib?

Or `clean`: for removing lib
and `clean:dependencies` for node_modules

Or `clean` for node_modules
and `clean:compiles` for lib?

Reply:
Clean removes node modules
precompile removes lib

Should clean also remove package-lock.json?